cfgov/preprocessed/js/modules/polyfill/class-list.js

```diff
@@ -0,0 +1,237 @@
+/*
+ * classList.js: Cross-browser full element.classList implementation.
+ * 2014-07-23
+ *
+ * By Eli Grey, http://eligrey.com
+ * Public Domain.
+ * NO WARRANTY EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.
+ */
+
+/*global self, document, DOMException */
+
+/*! @source http://purl.eligrey.com/github/classList.js/blob/master/classList.js*/
+
+if ("document" in self) {
+
+// Full polyfill for browsers with no classList support
+if (!("classList" in document.createElement("_"))) {
+
+(function (view) {
+
+"use strict";
+
+if (!('Element' in view)) return;
+
+var
+    classListProp = "classList"
```

Comment:
IMO we should move the polyfills to a npm module that we import like other 3rd-party code. We could edit it... but why? It shouldn't need to ever be touched since it targets legacy systems, and in touching it we may break some legacy edge-case quirk for syntax that isn't supported.